AdaGold/hello-books-api
# Update pip packages

Updates pip packages.

To test: 
*Clone repo to new location on local machine
* Switch branches to the final step (10c-nested-routes)
* Create venv
* Activate venv
* Install dependencies
* Ensure you have postgres running
* Create a db and test db, adding connection strings to .env file
* run app, add data, verify route handlers can use models without errors
* run pytest, verify no errors

Asana ticket: https://app.asana.com/0/1201700438643002/1202065410888111/f
